lib/FusionInventory/Agent/Task/Inventory/Generic/Firewall/Ufw.pm

```diff
@@ -0,0 +1,52 @@
+package FusionInventory::Agent::Task::Inventory::Generic::Firewall::Ufw;
+
+use strict;
+use warnings;
+
+use FusionInventory::Agent::Tools::Constants;
+use FusionInventory::Agent::Tools;
+
+sub isEnabled {
+    my (%params) = @_;
+    return 0 if $params{no_category}->{firewall};
```

Comment:
"return 0 if ..." not need as still present in FusionInventory::Agent::Task::Inventory::Generic::Firewall